### PR TITLE
Add required UserAgent HTTP header

### PR DESCRIPTION
**Pull request by CaseyHillers**
See https://docs.github.com/en/rest/overview/resources-in-the-rest-api?apiVersion=2022-11-28#user-agent-required

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+## 9.10.1
```
Comment (CaseyHillers) [inline CHANGELOG.md:1]: @kevmoo I bumped this off dev as need to update flutter/cocoon with this fix. Is this ready to be released, or should I keep it marked as dev?

Comment (kevmoo) [reply CHANGELOG.md:1]: I have no problem if you want to publish now.

**Review by kevmoo — APPROVED** [review]

**Comment by CaseyHillers** [comment]
Submitting with the red check on "Add PR labels on open / Unreleased." I filed https://github.com/SpinlockLabs/github.dart/issues/353 for tracking purposes but I think we'll need an admin to regenerate the token.

**Comment by robbecker-wf** [comment]
Released in version 9.8.0 https://github.com/SpinlockLabs/github.dart/releases/tag/9.8.0

**Pull request merged**